src/app/user-form/user-form.component.css

```diff
@@ -0,0 +1,65 @@
+.bvn{
+  text-align: center;
+}
+.bll{
+  text-align: center;
+  height: 39px;
+  width: 193px;
+  border-radius: 19px;
+  border: 1px solid black;
+  background-color: bisque;
+  margin-top: 20px;
+}
+.bnn{
+  text-align: center;
+  height: 39px;
+  width: 193px;
+  border-radius: 19px;
+  border: 1px solid black;
+  background-color: bisque;
+  margin-top: 20px;
+}
+
+.bnm{
+  text-align: center;
+  height: 42px;
+  width: 100%;
+  border-radius: 20px;
+  border: 1px solid black;
+  background-color: bisque;
+  margin-top: 20px;
+}
+
+
+.vbnm{
+  margin: 0 auto;/*отступы 1-высота.2-ширина*/
+  width: 200px ;/*ширина*/
+  margin-top: 90px;
+}
+
+.cvbnm{
+  border: 1px solid coral;
+  width: 400px;
+  height: 600px;
+  background-color: coral;
+  border-radius: 20px;
+  margin: 0 auto;
+}
+
+button[name="run_script"] {
+  text-align: center;
+  height: 39px;
+  width: 198px;
+  border-radius: 19px;
+  border: none;
+  padding: 10px 25px;
+  background: #ff9900;
+  cursor: pointer;
+  text-transform: uppercase;
+  font-weight: bold;
+  color: white;
+  margin-top: 20px;
+}
+button[name="run_script"]:hover {
```

Comment:
add class

Reply:
there are not need add class here. this code related to just button tag